Lib/Stack.hpp

```diff
@@ -848,6 +848,7 @@ class Stack
 
 
   friend std::ostream& operator<<(std::ostream& out, const Stack<C>& s) {
+    using namespace Kernel;
```

Comment:
?

Reply:
This slipped though.. had been needed before refactoring `Lib/Output.hpp` as we needed to explicitly import `Kernel::operator<<`... basically compilcated and uninteresting namespacing issues. I just removed it agin :)